examples/md/index.js

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
 import { render } from 'react-dom';
 
-import { Box, Deck, FlexBox, FullScreen, Markdown, Progress } from 'spectacle';
+import { Box, Deck, FlexBox, FullScreen, Progress } from 'spectacle';
 
 // SPECTACLE_CLI_MD_START
 import mdContent from './slides.md';
+import { MarkdownSlideSet } from '../../src/components/markdown/markdown';
```

Comment:
1. This will need to come off a root level `spectacle` re-export.
2. It cannot be within the `SPECTACLE_CLI_MD_START` + `SPECTACLE_CLI_MD_END` control comments. These get regex-replaced by the CLI for custom slides.